Assets/Scripts/MLA Pong/Agent/PongAgent.cs

```diff
@@ -19,6 +19,8 @@ public class PongAgent : Agent
         [Header("Others")]
         [SerializeField] GameObject manager;
 
+        [SerializeField] public bool isPlayer;
```

Comment:
Brauchst kein SerializeField und public zusammen. In public ist SerializeField ja bereits enthalten